README.md

```diff
@@ -40,54 +40,101 @@ git clone https://github.com/fortran-lang/stdlib
 cd stdlib
 ```
 
+
 ### Requirements
 
-The preprocessor ```fypp``` (https://github.com/aradi/fypp) is needed because metaprogramming is used.
-It can be installed using the command line installer ```pip```.
+To build the Fortran standard library you need
+
+- a Fortran 2008 compliant compiler, better a Fortran 2018 compliant compiler
+  (GCC Fortran or Intel Fortran compilers are known to work for stdlib)
+- CMake version 3.14 or newer (alternatively make can be used)
+- a build backend for CMake, like make or ninja (the latter is recommended on Windows)
+- the [``fypp``](https://github.com/aradi/fypp) preprocessor (used as meta-programming tool)
+
+If your system package manager does not provide the required build tools, all build dependencies can be installed with the Python command line installer ``pip``:
+
 ```sh
-pip install fypp
+pip install fypp cmake ninja
 ```
 
+Alternatively, you can install the build tools from the conda-forge channel with the conda package manager:
+
+```sh
+conda config --add channels conda-forge
+conda create -n stdlib-tools fypp cmake ninja
+conda activate stdlib-tools
+```
+
+You can install conda from using the [miniforge installer](https://github.com/conda-forge/miniforge/releases).
+Also, you can install a Fortran compiler from conda-forge by installing the ``fortran-compiler`` package, which installs ``gfortran``.
+
+
 ### Build with CMake
 
+Configure the build with
+
 ```sh
 cmake -B build
+```
+
+you can pass additional options to CMake to customize the build.
+Important options are
+
+- `-G Ninja` to use the ninja backend instead of the default make backend, other build backends are available with a similar syntax
```

Comment:
```suggestion
- `-G Ninja` to use the Ninja backend instead of the default Make backend. Other build backends are available with a similar syntax.
```